### ISSUE TITLE
train.py hasn't finished yet? especially for preprocessing.py in python_utils?

### ISSUE THREAD

**Issue by sainttelant** [Vladkryvoruchko/PSPNet-Keras-tensorflow#53]
as mentioned above, i found many places demanding to debug, i don;t know the datasets how to place in the folders by author. especially for the function of "data_generator_s31 " and function of  “generate”， it couldn't return the correct of images and corresponding annotations, i am trying to fix them up, does anyone find the same issues?

**Comment by xiaowoniuli** [comment]
> 
> 
> as mentioned above, i found many places demanding to debug, i don;t know the datasets how to place in the folders by author. especially for the function of "data_generator_s31 " and function of “generate”， it couldn't return the correct of images and corresponding annotations, i am trying to fix them up, does anyone find the same issues?

Have you resolved this problem? in the data_generator_s31 i can not find the label and don not know the src and label should to put.if  you train youself data please tell me  ,thanks
